feat: Show color badges for severity on reports lists

nublarsec opened the pull request:
This change adds color badges to the severity counts on the report list pages, making them easier to scan visually.

This resolves issue #83.

Vulnerability Reports page:
<img width="1329" alt="Screen Shot 2022-04-21 at 1 01 58 PM" src="https://user-images.githubusercontent.com/103554953/164551405-a2cc340a-a064-4c34-8bae-aeb1ba3a1447.png">

Config Audit Reports page:
<img width="1334" alt="Screen Shot 2022-04-21 at 1 54 18 PM" src="https://user-images.githubusercontent.com/103554953/164551429-2b4b6b4f-869a-4d27-a73d-e804d34f41cc.png">

CIS Kube Bench Reports page:
<img width="1336" alt="Screen Shot 2022-04-21 at 1 54 27 PM" src="https://user-images.githubusercontent.com/103554953/164551464-5d1fcc7b-c129-4a81-b4cc-c1642977e316.png">

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/aquasecurity/starboard-lens-extension?pullRequest=84) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/aquasecurity/starboard-lens-extension?pullRequest=84) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/aquasecurity/starboard-lens-extension?pullRequest=84) it.</sub>

danielpacak commented on src/configauditreports/page.scss:
```diff
@@ -0,0 +1,34 @@
+      background-color: #096ab0;
+    }
+
+    &.severity-NEGLIGIBLE {
+      color: white;
+      background-color: #00b09b;
+    }
+
+    &.severity-UNKNOWN {
+      color: white;
+      background-color: #515456;
+    }
```
For configuration audits we only have Critical, High, Medium, and Low severities.

danielpacak commented on src/vulnerabilityreports/page.scss:
```diff
@@ -0,0 +1,34 @@
+      background-color: #096ab0;
+    }
+
+    &.severity-NEGLIGIBLE {
+      color: white;
+      background-color: #00b09b;
+    }
```
We don't use Negligible severity in anymore.

danielpacak reviewed (changes requested):
Great idea @nublarsec and thanks for working on the implementation. I left minor comments to apply. I also noticed that the text is not exactly centered.

![lens_badges](https://user-images.githubusercontent.com/1322923/164679513-5e14659d-6b98-41ee-abd2-0e49884a1015.png)

nublarsec commented:
@danielpacak thanks for the quick reply. I've dropped the unused severity styles. 

I also changed the badges for zero counts so they still appear with the same size and shape. This looks better and keeps all the counts aligned.
